### PR TITLE
feat: adding "ingress" as a field to "service" in the runtime config. Not wired yet.

### PR DESCRIPTION
**Pull request by nicolasalt**

### CODE REVIEW

**Comment by nicolasalt** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1020** <a href="https://app.graphite.dev/github/pr/namespacelabs/foundation/1020" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/namespacelabs/foundation/1020?utm_source=stack-comment).

**Review by hugosantos — APPROVED** [review]
Please move Domain under Ingress, but otherwise LGTM.

File: schema/runtime/config.proto [hunk 1]
```diff
@@ -44,9 +44,19 @@ message Server {
     }
 
     message Service {
-        string owner    = 1;  // Package name.
-        string name     = 2;  // Scoped to package name.
-        string endpoint = 3;  // E.g. hostname:port.
+        string  owner    = 1;  // Package name.
+        string  name     = 2;  // Scoped to package name.
+        string  endpoint = 3;  // E.g. hostname:port.
+        Ingress ingress  = 4;  // Computed ingress for this service.
```
[the remark this answers is not in the document]
Comment (nicolasalt) [reply schema/runtime/config.proto:50]: Maybe "repeated" then?

File: schema/runtime/config.proto [hunk 2]
```diff
@@ -44,9 +44,19 @@ message Server {
...
+    }
+
+    message Ingress {
+        repeated Domain domain = 4;
+    }
+
+    message Domain {
```
[the remark this answers is not in the document]
Comment (nicolasalt) [reply schema/runtime/config.proto]: Done.

**Pull request merged**